rust/private/rustc.bzl

```diff
@@ -584,6 +584,41 @@ def collect_inputs(
 
     return compile_inputs, out_dir, build_env_files, build_flags_files, linkstamp_outs, ambiguous_libs
 
+def _collect_ld_library_path(env, rust_toolchain, cc_toolchain):
+    """Construct the value of `LD_LIBRARY_PATH` based on the existing environment and registered toolchains.
+
+    Args:
+        env (dict): A dict of existing environment variables
+        rust_toolchain (rust_toolchain): The registered rust_toolchain
+        cc_toolchain (CcToolchainInfo): A cc toolchain provider to be used.
+
+    Returns:
+        str: The value of `LD_LIBRARY_PATH`.
+    """
+    sep = ";"
+    ld_paths = []
+
+    # Maintain existing values
+    if "LD_LIBRARY_PATH" in env:
+        ld_paths.extend(env["LD_LIBRARY_PATH"].split(sep))
+    if "DYLD_LIBRARY_PATH" in env:
+        ld_paths.extend(env["DYLD_LIBRARY_PATH"].split(sep))
+
+    # Add the rustc sysroot
+    ld_paths.append("{}/lib".format(rust_toolchain.sysroot))
+
+    # When possible add the cc sysroot
+    if cc_toolchain.sysroot:
```

Comment:
This part seems like is a part of the C++ toolchain configuration. I know that sysroot handling in the C++ toolchain is not great, do you use https://docs.bazel.build/versions/main/command-line-reference.html#flag--grte_top? If not, just hardcode all the flags in the CcToolchainConfigInfo and ignore there is a builtin support for sysroot in Bazel :)

Reply:
Was this comment understandable? Does it make sense?

Reply:
(sorry, missed this. 😅)

I do not use `--grte_top` but based on your earlier comments about toolchains defining sufficient flags and variables for building, I would say this should be removed and will do so.